### ISSUE TITLE
[bug] Failed to build datasets for 2 species with option --include_genus

### ISSUE THREAD

**Issue by llk578496** [bactopia/bactopia#316]
**Description**
We tried to build datasets for 2 species at one time with the option --include_genus, but it fails.

**Steps to Reproduce**
```
bactopia datasets --species 'Enterococcus faecium, Enterococcus faecalis' --include_genus --cpus 10 --outdir ${AWD}/00_ref/datasets
```

**Expected Behavior**
The Bactopia documentation also used 2 species for building the datasets.
`bactopia datasets --species "Haemophilus influenzae,Staphylococcus aureus" --include_genus`

**Execution Environment**

- Bactopia Version: 2.0.3
- OS: Ubuntu 20.04.4
- Environment: conda

**Additional Information**
Thank you very much for developing this amazing pipeline

**Comment by rpetit3** [comment]
Hi @llk578496 

I need to update the docs on this. `--include_genus` can only be used with a single species at a time.  But, I'm going  to see if my 2022-self can make what my 2019-self couldn't get to work. 

Will update soon,
Robert

**Comment by llk578496** [comment]
Hi @rpetit3 ,

Thank you.

Please don't give up! 

`Bactopia` is one of the most amazing pipeline we have ever seen in the field of microbiology.

We will definitely wait for your update!

**Comment by rpetit3** [comment]
Thank you very much for your kind words! 

Please keep providing feedback so we can keep making it better!

**Comment by rpetit3** [comment]
Took a look at this, and it was a straight-forward change. I have a developement build on my personal conda channel, if you are interested in trying.

```
mamba install -n bactopia-dev -c rpetit3 -c conda-forge -c bioconda bactopia
```

**Comment by rpetit3** [comment]
Hi @llk578496 

I released v2.1.0 (https://github.com/bactopia/bactopia/releases/tag/v2.1.0) which should fix includes the changes from the dev build you were using. If your problem persists, please feel free to reopen this issue!

If you have some time to spare, please consider filling out my Bactopia survey.  I use the feedback to direct the future developments of Bactopia. (_If you've already filled it out, thank you so much!_)

https://forms.gle/PTgUxEBwCsCdbQjr7

Cheers,
Robert
